sdk/services/futures.ts

```diff
@@ -272,6 +272,32 @@ export default class FuturesService {
 		return positions;
 	}
 
+	public async getMarketFundingRates(marketAsset: FuturesMarketAsset) {
+		const periodLength = PERIOD_IN_SECONDS.TWO_WEEKS;
+		const minTimestamp = Math.floor(Date.now() / 1000) - periodLength;
```

Comment:
Probably best we pass this in as a param just in anticipation of being flexible for others using this call once the sdk becomes public.

Reply:
Done